src/test/java/org/codehaus/plexus/classworlds/ClassWorldTest.java

```diff
@@ -28,35 +28,35 @@
 import org.codehaus.plexus.classworlds.realm.ClassRealm;
 import org.codehaus.plexus.classworlds.realm.DuplicateRealmException;
 import org.codehaus.plexus.classworlds.realm.NoSuchRealmException;
-import org.junit.After;
-import org.junit.Before;
-import org.junit.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 public class ClassWorldTest
```

Comment:
`public`  from class can also be removed

Reply:
fixed thx.